src/nvim/edit.c

```diff
@@ -3103,7 +3103,7 @@ void get_complete_info(list_T *what_list, dict_T *retdict)
   }
 
   if (ret == OK && (what_flag & CI_WHAT_SELECTED)) {
-    ret = tv_dict_add_nr(retdict, S_LEN("selected"),
+    tv_dict_add_nr(retdict, S_LEN("selected"),
```

Comment:
because of the TODO on line 3111, i would instead add 

    (void)ret;

after this if-block. Otherwise we'll be confused when merging Vim's TODO changes later.